l3build-check.lua

```diff
@@ -551,6 +551,11 @@ local function normalize_pdf(content)
       binary = false
       stream = true
       stream_content = "stream" .. os_newline
+    elseif match(line, "/Length %d+>>stream$") then
```

Comment:
Maybe `">>stream$"` is enough, but I chose a more strict pattern, mainly to not match a non-stream.